client/ayon_traypublisher/plugins/create/create_texture.py

```diff
@@ -30,7 +30,9 @@ class TextureCreator(TrayPublishCreator):
     icon = "fa.file"
     description = "Texture files or UDIM sequences"
     extensions = [".mov", ".mp4", ".mxf", ".m4v", ".mpg", ".exr",
-                  ".dpx", ".tif", ".png", ".jpg"]
+                  ".dpx", ".tif", ".png", ".jpg", "tga", "tx"]
+
+    default_variants = ["_col", "_spec", "_nrm", "_disp", "_rough", "_metel", "_sss", "_mask", "_id "]
```

Comment:
I think this is specific to your workflow and shouldn't be forced upstream.
Also, these are currently configurable in settings already:
```
ayon+settings://traypublisher/create/TextureCreator/default_variants
```
So these can be removed hardcoded here and moved into your settings.


```suggestion
```